### PR TITLE
feat: implement vc details component

### PR DESCRIPTION
**Pull request by anusha-kommu**
desktop view:
Screenshot 2023-02-13 at 15 43 11

mobile view:
Screenshot 2023-02-13 at 15 43 17

Breaking text into new line if text content is too big for container:
Screenshot 2023-02-13 at 15 43 30

### CODE REVIEW

File: pages/holder/components/VcDetails/VcDetails.styled.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    margin-bottom: ${pxToRem(40)};
+
+    img {
+      height: ${pxToRem(292)};
+    }
+  }
+`
+
+export const VcDetailsContainer = styled(Box)`
+  flex-direction: row;
```
Comment (OleksiiO-affinidi) [inline pages/holder/components/VcDetails/VcDetails.styled.ts:28]: use `direction` as prop for `Box`

File: pages/holder/components/VcDetails/VcDetails.styled.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  }
+
+  @media (min-width: 1024px) {
+    border-top-right-radius: 40px;
+    border-bottom-right-radius: 40px;
+  }
+`
+
+export const VcDetailsCard = styled(Box)`
+  gap: ${pxToRem(48)};
```
Comment (OleksiiO-affinidi) [inline pages/holder/components/VcDetails/VcDetails.styled.ts:45]: use `gap` as prop for `Box`

File: pages/holder/components/VcDetails/VcDetails.styled.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    word-wrap: break-word;
+  }
+
+  @media (max-width: 1024px) {
+    gap: ${pxToRem(36)};
+  }
+`
+
+export const MedicationDetailsContainer = styled(Box)`
+  gap: ${pxToRem(42)};
```
Comment (OleksiiO-affinidi) [inline pages/holder/components/VcDetails/VcDetails.styled.ts:58]: the same

File: pages/holder/index.tsx [hunk 4]
```diff
@@ -75,43 +70,36 @@ const Home: FC = () => {
...
-  const getPrescriptionCards = ({
-    prescriptions,
-    isValid,
-  }: {
-    prescriptions: StoredW3CCredential[]
-    isValid: boolean
-  }) =>
-    prescriptions.map((credentialItem: StoredW3CCredential) => {
+  const getVcCards = ({ vcs }: { vcs: StoredW3CCredential[] }) => {
+    let vcNumber = 1
```
Comment (OleksiiO-affinidi) [inline pages/holder/index.tsx:80]: why do u need this `let`?

**Review by ezgi-affinidi — APPROVED** [review]

**Pull request merged**